src/Components/NavBar.test.js

```diff
@@ -0,0 +1,18 @@
+import { shallow } from "enzyme";
+import React from "react";
+
+import { NavBar } from "./NavBar";
+
+describe("NavBar", () => {
+  test.each([[0, "What is Pursuit?", 1, "Create an account", 2, "Sign in"]])(
```

Comment:
Does this need to be a 2d array of 3 elements?
```
test.each([
  [0, "What is Pursuit?"], 
  [1, "Create an account"], 
  [2, "Sign in"]
]( ...
```
Currently it looks like a 2d array with only 1 element

Reply:
Ooh nice catch...